lib/ord.js

```diff
@@ -95,7 +98,9 @@ const _getPackages = (policyLevel, appConfig) =>
             .slice(0, 1);
 
 const _getAPIResources = (csn, appConfig, packageIds) => {
+    if (appConfig.odmEntities.length === 0) return [];
     return appConfig.serviceNames
```

Comment:
I am missing a filtering based on a service containing only events and no entity or functions or action

Reply:
> I am missing a filtering based on a service containing only events and no entity or functions or action

The logic is the following(pls check line 103): 
Some service is considered an `apiResource` _only_ if:
1. there is at least one entity existing, with **name starting with** that-service-name &&
2. the name of that entity **does not include** string `.texts`. 

Similarly, some service is considered an `eventResource` _only_ if:
1. there is at least one event existing, with **name starting with**  that-service-name

This is expressed by the lines from below. @swennemers @Fannon  : Do you agree?
`
            // TODO: should be rewritten
            case ORD_RESOURCE_TYPE.entity:
                if (!key.includes(".texts")) {
                    entities.add(key); // I've introduced the entities Set
                    if (keyDefinition["@ODM.entityName"]) {
                        odmEntities.push(createEntityTypeTemplate(keyDefinition));
                    }
                }
                break;
            case ORD_RESOURCE_TYPE.event:
                events.push(key);
                break;
`

`
.filter((serviceName) => appConfig.entities.some((entityName) => entityName.startsWith(serviceName)))
`

Reply:
I am missing functions + actions, that result in APIs. I don't get what this has to do with a variable called odmEntities or a  case ORD_RESOURCE_TYPE.entity.